lesson3/src/style.css

```
@@ -0,0 +1,158 @@
+body {
+  height: 100vh;
+  font-family: Arial, Helvetica, sans-serif;
+  background-color: rgb(23 23 23 / 100%);
+}
+
+.header {
+  display: flex;
+  flex-wrap: nowrap;
+  align-items: center;
+  justify-content: space-between;
+  height: 70px;
+  background-color: rgb(31 31 31 / 100%);
+}
+
+article span {
+  width: 260px;
+  height: 120px;
+  padding: 80px;
+  text-align: center;
+  background-color: rgb(41 41 41 / 100%);
+}
+
+.header span {
+  display: flex;
+  flex-wrap: nowrap;
+  justify-content: flex-start;
+  padding-left: 50px;
+  font-size: 20px;
+  color: #fff;
+}
+
+.nav {
+  display: flex;
+  flex-wrap: nowrap;
+  gap: 20px;
+  justify-content: flex-end;
+  padding-right: 50px;
+  font-size: 20px;
+  color: #fff;
+}
+
+a:hover {
+  color: rgb(43 155 253 / 100%);
+}
+
+.container {
+  display: flex;
+  flex-flow: row wrap;
+  column-gap: 50px;
+  align-items: center;
+  justify-content: center;
+  padding: 50px;
```

```suggestion
  display: flex;
  flex-flow: row nowrap;
  column-gap: 48px;
  max-width: 1140px;
  margin: 0 auto;
```

- для контейнера main можно задать максимальную ширину и центрирование через margin 0 auto
- отступ в макете 48px